tests/test_modeling_beit.py

```diff
@@ -162,11 +162,11 @@ def create_and_check_for_image_segmentation(self, config, pixel_values, labels,
         model.eval()
         result = model(pixel_values)
         self.parent.assertEqual(
-            result.logits.shape, (self.batch_size, self.num_labels, self.image_size, self.image_size)
+            result.logits.shape, (self.batch_size, self.num_labels, self.image_size * 2, self.image_size * 2)
```

Comment:
This is really weird @NielsRogge . Why does BeIT return a bigger size with image_size=30?